Package.swift

```diff
@@ -12,27 +12,37 @@ let package = Package(
             name: "MapboxMaps",
             targets: ["MapboxMaps"]),
     ],
+    // when upgrading to swift tools version 5.6 or later,
+    // replace `.package(name:url:_: .exact())` with `.package(url:exact:)`
     dependencies: [
-        .package(name: "MapboxCoreMaps", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.5.1")),
-        .package(name: "MapboxCommon", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("21.3.0")),
-        .package(name: "MapboxMobileEvents", url: "https://github.com/mapbox/mapbox-events-ios.git", .exact("1.0.7")),
-        .package(name: "Turf", url: "https://github.com/mapbox/turf-swift.git", from: "2.0.0"),
-        .package(name: "CocoaImageHashing", url: "https://github.com/ameingast/cocoaimagehashing", .exact("1.9.0"))
+        .package(name: "mapbox-core-maps-ios", url: "https://github.com/mapbox/mapbox-core-maps-ios.git", .exact("10.5.1")),
+        .package(name: "mapbox-common-ios", url: "https://github.com/mapbox/mapbox-common-ios.git", .exact("21.3.0")),
+        .package(name: "mapbox-events-ios", url: "https://github.com/mapbox/mapbox-events-ios.git", .exact("1.0.7")),
```

Comment:
I reverted back to the deprecated methods here since the replacement for `.exact` isn't available until Swift 5.6. Initially I used revision and the tag name instead, but that doesn't play nicely with depsvalidator.